x-pack/solutions/security/packages/expandable-flyout/src/components/resizable_container.tsx

```diff
@@ -77,38 +80,43 @@ export const ResizableContainer: React.FC<ResizableContainerProps> = memo(
     );
 
     return (
-      <EuiResizableContainer
-        css={css`
-          height: 100%;
-        `}
-        onPanelWidthChange={onWidthChange}
-      >
-        {(EuiResizablePanel, EuiResizableButton) => (
-          <>
-            <EuiResizablePanel
-              id={LEFT_PANEL_ID}
-              initialSize={initialLeftPercentage}
-              size={leftPercentage}
-              minSize={LEFT_SECTION_MIN_WIDTH}
-              paddingSize="none"
-              data-test-subj={RESIZABLE_LEFT_SECTION_TEST_ID}
-            >
-              <LeftSection component={leftComponent} />
-            </EuiResizablePanel>
-            <EuiResizableButton disabled={showPreview} data-test-subj={RESIZABLE_BUTTON_TEST_ID} />
-            <EuiResizablePanel
-              id={RIGHT_PANEL_ID}
-              initialSize={initialRightPercentage}
-              size={rightPercentage}
-              minSize={RIGHT_SECTION_MIN_WIDTH}
-              paddingSize="none"
-              data-test-subj={RESIZABLE_RIGHT_SECTION_TEST_ID}
-            >
-              <RightSection component={rightComponent} />
-            </EuiResizablePanel>
-          </>
-        )}
-      </EuiResizableContainer>
+      <>
```

Comment:
dont think you need the empty tag here?

Reply:
that is correct, this is left over from some test I was doing with a resize button. Removed!